File: 00-user-defined-data-type.js
class client {

    constructor(Name = '', Id = '', Credit = 0, Address = '') {
        this.Name = Name;
        this.Id = Id;
        this.Credit = Credit;
        this.Address = Address;
    }

}

const client1 = new client();

client1.Name = "Sebastia Rodriguez"
client1.Id = "000000001"
client1.Credit = 1_000_000
client1.Address = "Cll 123 # 45 - 67"

console.log(`The cliente name is: ${client1.Name}`);
console.log(`The cliente id is: ${client1.Id}`);
console.log(`The cliente credit is: ${client1.Credit}`);
console.log(`The cliente address is: ${client1.Address}`);

File: 00-user-defined-data-type.c
#include "stdlib.h"
#include "stdio.h"
#include "string.h"

struct client
{
    char Name[50];
    char Id[10];
    float Credit;
    char Address[100];
};

int main(int argc, char const *argv[])
{
    struct client client1 = {0};

    strcpy(client1.Name, "Sebastian Rodriguez Marin");
    strcpy(client1.Id, "000000001");
    strcpy(client1.Address, "Cll 123 # 45 - 67");
    client1.Credit = 1000000;

    printf("The client name is: %s\n", client1.Name);
    printf("The client id is: %s\n", client1.Id);
    printf("The client address is: %s\n", client1.Address);
    printf("The client credit is: %f\n", client1.Credit);

    return 0;
}

